libs/types/src/errors.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Possible errors that can occur during configuration (currently there's only one).
+ */
+export type ConfigurationError = 'no_ballot_package_on_usb_drive';
```

Comment:
I think that `ConfigurationError` in `libs/types` is pretty generic naming, but this is actually a specific type of configuration. Could it have a more descriptive name? Alternatively I think it would be fine to colocate it in `libs/ui` if that's where it is being used, but this works too.

Reply:
I ported this as-is but I definitely agree - updated to `BallotPackageConfigurationError` in [b62b061](https://github.com/votingworks/vxsuite/pull/3156/commits/b62b061ba2914dcc2e0e7261a5de603cb0d18517)